eslintrc.json

```diff
@@ -336,7 +336,7 @@
     "no-unused-vars": [
       "error",
       {
-        "varsIgnorePattern": "apos",
+        "varsIgnorePattern": "^_",
```

Comment:
we have variables with leading underscore unused and that's ok?

Reply:
I don't know why this is here.

Reply:
i think because some params could be unused, and we wanna have a way to mark them as "not used but that's ok":
.

Ex:

```js
function hey(_unused1, _unused2, foo) {
  // ...
}
```

also for arrays:

```js
const [ _unused1, foo, _unused3, bar ] = array;
```

even thought we can write:

```js
const [ , foo, , bar ] = array;
```

right @ValJed?

Reply:
I would be OK with moving to the skipped arguments syntax:

```
const [ , foo, , bar ] = array;
```

But that would take more updating in the short term. Let's do what gets us to consistency soonest on this.

Reply:
Unused arguments were probably never covered by this rule, because I know we often take `req` for the sake of subclasses that might override a method later.

Reply:
I think this concerns only unused variables and not arguments. Unused arguments did never show as a problem and they shouldn't. Unused vars were always marked as an `error`. 

I think this rule can be useful when destructing an object:  
```js
const { ignoreMe1: _ignore1, ignoreMe2: _ignore2, ...rest } = obj;
// do something with `rest`
```

Reply:
This rule is useful for function params, but as stated Miro we mainly use if for object destructuring, it should be kept for this.